java-checks/src/main/java/org/sonar/java/checks/security/FileHandlingCheck.java

```diff
@@ -49,6 +49,7 @@
       MethodMatcher.create().typeDefinition(JAVA_IO_FILE).name(INIT).parameters(JAVA_LANG_STRING),
       MethodMatcher.create().typeDefinition(JAVA_IO_FILE).name(INIT).parameters(JAVA_LANG_STRING, JAVA_LANG_STRING),
       MethodMatcher.create().typeDefinition(JAVA_IO_FILE).name(INIT).parameters("java.net.URI"),
+      MethodMatcher.create().typeDefinition(JAVA_IO_FILE).name("createTempFile").parameters(JAVA_LANG_STRING, JAVA_LANG_STRING),
```

Comment:
I guess we don't consider the other `createTempFile()` having a file as parameter for the same reason as file constructor: because the file would already have been flagged?